tox.ini

```diff
@@ -15,6 +15,7 @@ commands =
   {toxinidir}/terraform_test_artifacts/terraform_install.sh
   find . -type f -name "*.pyc" -delete
   pytest {posargs}
+allowlist_externals = {toxinidir}/*,bash,find
 whitelist_externals =
   bash
   find
```

Comment:
Since we're using `allowlist_externals` - I believe we could now remove the `whitelist_externals`.
https://tox.wiki/en/latest/config.html#allowlist_externals
Since I think, `whitelist_externals` has been deprecated, but not 100% sure :sweat_smile: :smile: 
Since if we leave both, I'm not sure if one configuration overrides the other.